docs/INSTALLATION.md

```diff
@@ -1,17 +1,27 @@
 Please reference Racket's [Getting Started](http://docs.racket-lang.org/getting-started/) page for instructions on downloading and installing Racket.
 
-For *nix variants, you can also install Racket using your OS' package manager:
+Alternative installation options and further notes for specific OSes follow.
 
-**OS X**
+## OS X
 
 Install via Homebrew:
 
 ```
 brew update
+brew install minimal-racket
+```
+The `minimal-racket` package will probably wor for solving the problems, but it does not include the graphical interface IDE, DrRacket. If you want that, you have to install the full racket distribution from cask:
+
+```
+brew tap caskroom/cask
 brew install racket
 ```
 
-**Linux**
+### Note on using the official installer on OS X
```

Comment:
Maybe this could be shortened to something like Official macOS installer. We should probably replace "OS X" to macOS throughout to match Apple's (re)branding.